server/services/historicalPrisonerService.ts

```diff
@@ -18,27 +19,33 @@ export default class HistoricalPrisonerService {
   async findPrisonersByName(
     token: string,
     prisonersByNameForm: FindPrisonersByName,
+    page: number = undefined,
   ): Promise<PagedModelPrisonerSearchDto> {
     return HistoricalPrisonerService.restClient(token).get<PagedModelPrisonerSearchDto>({
       path: '/search',
-      query: querystring.stringify({ ...prisonersByNameForm, size: 10 }),
+      query: querystring.stringify({ ...prisonersByNameForm, page }),
     })
   }
 
   async findPrisonersByIdentifiers(
     token: string,
     prisonersByIdentifiersForm: FindPrisonersByIdentifiers,
+    page: number = undefined,
   ): Promise<PagedModelPrisonerSearchDto> {
     return HistoricalPrisonerService.restClient(token).get<PagedModelPrisonerSearchDto>({
       path: '/identifiers',
-      query: querystring.stringify({ ...prisonersByIdentifiersForm }),
+      query: querystring.stringify({ ...prisonersByIdentifiersForm, page }),
     })
   }
 
-  async findPrisonersByAddressTerms(token: string, addressTerms: string): Promise<PagedModelPrisonerSearchDto> {
+  async findPrisonersByAddressTerms(
+    token: string,
+    prisonersByAddress: FindPrisonersByAddress,
+    page: number = undefined,
```

Comment:
wasn't sure what the "norm" was - whether to set undefined or null (same for form fields)

Reply:
don't know either 😁